python/pyarrow/_dataset.pyx

```diff
@@ -1440,7 +1441,6 @@ cdef class DirectoryPartitioning(Partitioning):
             res.append(pyarrow_wrap_array(arr))
         return res
 
-
```

Comment:
I think we need to keep two spaces between class definitions as per [PEP 8](https://peps.python.org/pep-0008/#blank-lines).

Reply:
Made the change.